projects/tracker/schematics/version.ts

```diff
@@ -1 +1 @@
-export const version = '^3.2.0';
+export const version = '^4.0.0';
```

Comment:
Version will be bumped automatically when merged, so it should not be manually changed.